### DISCUSSION TITLE
Add support for json feeds.

### DISCUSSION THREAD

**Discussion by TheOneImprove** [jocmp/capyreader#373] [Feature Requests]
https://www.jsonfeed.org/

**Comment by jocmp** [comment]
@TheOneImprove any feeds you have in mind? I'll use those to help build out this feature. It's something I can add now since I rewrote the parser.

**Reply by TheOneImprove** [reply]
- **NPR**: https://feeds.npr.org/1004/feed.json

**Blogs:**

- https://kibty.town/blog.json

**Reply by jocmp** [reply]
thanks for these. Tracking via https://github.com/jocmp/capyreader/issues/383

**Comment by jocmp** [comment]
Added as of 2024.09.1056-dev
